src/main/resources/EmailService.properties

```diff
@@ -1,10 +1,17 @@
 email.use.aws=${USE_AWS_SES}
 sender.email.address=${EMAIL_ADDRESS}
 
+<<<<<<< Updated upstream
```

Comment:
This file is not merged. Conflicts still present

Reply:
Made the changes